src/components/views/styl/Header.styl

```diff
@@ -96,27 +95,27 @@ h1 a:hover {
     position: absolute;
 }
 
-.navigation-group:hover .subnav {
-    display: flex;
-    flex-direction: column;
-
-    left: -20px; /* Equal to the left-padding of the li, so that text aligns */
-}
-
 .navigation-coming-soon:hover .subnav {
     display: block;
     padding: 15px;
     left: -15px; /* Equal to the left-padding so that text aligns */
 }
 
-.navigation-group:hover li {
-    padding: 8px 20px;
-}
+.navigation-group:hover {
+    .subnav {
+        display: flex;
+        flex-direction: column;
 
-.navigation-group:hover li:first-child {
-    padding-top: 16px; /* 2x the normal padding */
-}
+        left: -20px; /* Equal to the left-padding of the li, so that text aligns */
+    }
+    li {
+        padding: 8px 20px;
```

Comment:
Another case where a variable might help. I'm sorry I didn't notice this before.